cumulusci/tasks/bulkdata/delete.py

```diff
@@ -51,11 +56,24 @@ def _create_job(self, obj):
         self.bulk.close_job(delete_job)
         return delete_job
 
-    def _query_salesforce_for_records_to_delete(self, obj):
+    def compose_query(self, obj, criteria):
```

Comment:
Should this be `_compose_query()`?

Reply:
Great question!

I actually decided to do it this way to enable sub-classing so I consider this part of the public interface in case you wanted to do something more sophisticated than a WHERE clause.

"Joins" in particular, I think.